db/migrate/20150107044854_users_without_location_cleanup.rb

```diff
@@ -0,0 +1,9 @@
+class UsersWithoutLocationCleanup < ActiveRecord::Migration
+  def up
+    User.delete_all(location: [nil,''])
```

Comment:
What do you think the cause for the issue then? definitely there is something mandate that is missing for a user row.. It was prescribed as `location` so made this fix..